R/pkg/R/generics.R

```diff
@@ -396,67 +397,81 @@ setGeneric("agg", function (x, ...) { standardGeneric("agg") })
 #' @param object x a SparkDataFrame or a Column
 #' @param data new name to use
 #' @return a SparkDataFrame or a Column
+#' @noRd
 NULL
 
 #' @rdname arrange
 #' @export
+#' @noRd
 setGeneric("arrange", function(x, col, ...) { standardGeneric("arrange") })
 
 #' @rdname as.data.frame
 #' @export
+#' @noRd
 setGeneric("as.data.frame",
            function(x, row.names = NULL, optional = FALSE, ...) {
              standardGeneric("as.data.frame")
            })
 
 #' @rdname attach
 #' @export
+#' @noRd
 setGeneric("attach")
 
 #' @rdname cache
 #' @export
+#' @noRd
 setGeneric("cache", function(x) { standardGeneric("cache") })
 
 #' @rdname checkpoint
 #' @export
+#' @noRd
 setGeneric("checkpoint", function(x, eager = TRUE) { standardGeneric("checkpoint") })
 
 #' @rdname coalesce
 #' @param x a Column or a SparkDataFrame.
```

Comment:
see here https://github.com/apache/spark/pull/18025/files#diff-04c14efaae2b7b0f0a45038482f2590cR135
how do we decide this goes to Column.R and not DataFrame.R? It's very easy then later on someone else added more comment in DataFrame.R thinking there is no documentation and then later on the help content is duplicated (has happened a few times before)